flixel/FlxCamera.hx

```diff
@@ -1635,7 +1638,7 @@ class FlxCamera extends FlxBasic
         }
 
         filters.remove(filter);
-    }
+    } */
```

Comment:
actually remove this code, don't just comment it out

Reply:
Alright